spec/api/v1/admin/patients_spec.rb

```diff
@@ -0,0 +1,162 @@
+require 'rails_helper'
+require 'active_record'
+# rubocop:disable RSpec/MultipleExpectations
+RSpec.describe V1::Admin::Patients, type: :request do
+  let(:patients_crud_url) { '/api/v1/admin/patients' }
+  let(:patients_params) do
+    {
+      address: Faker::Address.full_address,
+      date_of_birth: Faker::Date.between(from: '1900-01-01', to: Time.zone.today),
+      email: Faker::Internet.email,
+      full_name: Faker::Name.name,
+      notes: 'Something notes',
+      telephone_number: Faker::Number.number(digits: 7),
+      passport_id: Faker::Alphanumeric.alpha(number: rand(6..50)),
+      allergies_additional: 'Something information about allergies',
+      allergies: ''
+    }
+  end
+
+  before do
+    create_list :patient, 5
```

Comment:
try to not skip brackets `(` since it's a good practice to use them, especially in cases when you have to pass more than 1 param

look at [ruby style guide](https://github.com/rubocop/ruby-style-guide#method-call-parentheses-)